modules/cockroachdb/certs.go

```diff
@@ -65,3 +66,20 @@ func NewTLSConfig() (*TLSConfig, error) {
 		ClientKey:  clientCert.KeyBytes,
 	}, nil
 }
+
+// tlsConfig returns a [tls.Config] for options.
+func (c *TLSConfig) tlsConfig() (*tls.Config, error) {
+	keyPair, err := tls.X509KeyPair(c.ClientCert, c.ClientKey)
+	if err != nil {
+		return nil, fmt.Errorf("x509 key pair: %w", err)
```

Comment:
lint: I think we need to gofmt this file, as the `fmt` package is not imported. Please run `make lint` for this module.

Reply:
Looks like the auto merge messed things up.